src/etc/inc/interfaces.inc

```diff
@@ -1090,7 +1090,8 @@ function interface_bring_down($interface = "wan", $ifacecfg = false)
     switch ($ifcfg['ipaddrv6']) {
         case 'slaac':
         case 'dhcp6':
-            killbypid("/var/run/dhcp6c_{$realifv6}.pid", 'TERM', true);
+            interface_dhcpv6_prepare($interface, $ifcfg, true);
+            killbypid('/var/run/dhcp6c.pid', 'HUP');
```

Comment:
HUP works as expected now, rereading the interfaces from the configuration as well as command line. In this proposal we remove the command line use as well.

Reply:
I've just been driving myself mad trying to find out why the code would not start dhcp6c, it was because I had used a kill -QUIT command to shutdown dhcp6c and not a SIGTERM. QUIT behaves like -9 and leaves the PID behind preventing dhcp6c from starting. This is one of the reasons I added the checks and double checks to make sure this could not happen.. obviously I must have been thinking of myself! We really do need to make sure this cannot happen. If dhcp6c barfs for any reason then it will not restart until the pid is removed.

Reply:
I don't think we ever had problems again, but now that I know the issue it would make sense to prep for this case, i.e. stale pid file => remove it. we shouldn't assume there is another dhcp6c running. better yet, we can lock the start procedure so that only one script will ever be able to start it.

Reply:
Hence my original PR that checked for a stale PID and also checked the PID matched the process. I know it may have appeared as a total overkill but this dates back to pf*** and the same issues there. 

Even with just one script to start it you still need to make it foolproof... and I am that fool, if it can be broken, I WILL break it. ;)

Reply:
I've put back my rtsold script routine and it works well. Is there a reason we cannot use it?

Reply:
pgrep can return multiple PIDs, the check isn't so great. I'm pushing a different proposal for now:

check if the pid in the file is responding, if not delete pid file